packages/models/src/agreement/agreement.ts

```diff
@@ -39,7 +39,7 @@ export type AgreementDocument = z.infer<typeof AgreementDocument>;
 
 export const AgreementStamp = z.object({
   who: UserId,
-  delegateId: TenantId.optional(),
+  delegateProducerId: TenantId.optional(),
```

Comment:
About this and about the `delegateId` in the verified attributes see last message on Slack thread

Reply:
Solved in [this](https://github.com/pagopa/interop-be-monorepo/pull/1239/commits/ad13c6025c4e60d76c7e2541f426a9c16c55ce1c) commit